README.md

```diff
@@ -295,6 +295,8 @@ Use the "Table of Contents" menu on the top-left corner to explore the list.
 - [vite-plugin-zip-pack](https://github.com/7th-Cyborg/vite-plugin-zip-pack) - Pack distribution/build folder into a zip file.
 - [vite-plugin-cp](https://github.com/fengxinming/vite-plugins/tree/main/packages/vite-plugin-cp) - Copy files after building bundles.
 - [unplugin-compression](https://github.com/KeJunMao/unplugin-compression) - Compress dist to zip, tar, taz.
+- [vite-plugin-image-optimizer](https://github.com/FatehAK/vite-plugin-image-optimizer) - Optimize (compress) your image
+  assets using Sharp.js and SVGO at build time.
```

Comment:
new line ?

Reply:
Just fixed it and resolved conflicts. Please verify and merge.